index.js

```diff
@@ -347,7 +352,7 @@ BlinkDiff.prototype = {
 	 * @param {function} fn
 	 */
 	run: function (fn) {
-
+		console.log(this._blockOutOutputOpacity);
```

Comment:
Is this console.log needed for anything outside of debugging? If not, please remove